erpnext/accounts/utils.py

```
@@ -769,6 +771,32 @@ def update_reference_in_payment_entry(
+			"status": ["in", ["Initiated", "Payment Ordered"]],
+		},
+		"name",
+	)
+
+	if payment_request:
+		row.payment_request = payment_request
+		frappe.msgprint(
+			_(
+				"Payment Request {} has been allocated to Payment Entry {} at row #{}".format(
+					payment_request, row.parent, row.idx
+				)
```

Ruff (0.12.2)

Use f-string instead of `format` call

Convert to f-string

(UP032)